templates/corosync.conf.erb

```diff
@@ -56,6 +58,15 @@ aisexec {
 <% if @set_votequorum -%>
 quorum {
   provider: corosync_votequorum
+<% if @votequorum_expected_votes -%>
```

Comment:
I find this section a little hard to read. I think I understand the logic, but maybe the if statements around quorum and nodelist could be separated to make it more readable? Maybe it's just me though :-)